packages/beacon-node/src/api/rest/base.ts

```diff
@@ -97,7 +97,12 @@ export class RestApiServer {
       if (err instanceof ErrorAborted || err instanceof NodeIsSyncing) return;
 
       const {operationId} = res.context.config as RouteConfig;
-      this.logger.error(`Req ${req.id} ${operationId} error`, {}, err);
+
+      if (err instanceof ApiError && err.statusCode < 500) {
```

Comment:
consider just logging all api errors as warning

```suggestion
      if (err instanceof ApiError) {
```

Reply:
I'm in favor of this change